spec/high_voltage/configuration_spec.rb

```diff
@@ -61,7 +74,20 @@
     end
   end
 
-  describe '#page_caching=' do
+  describe '#action_caching_layout=false' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.